packages/blocks/src/store/selectors.js

```diff
@@ -349,6 +738,30 @@ export const hasChildBlocks = ( state, blockName ) => {
  * @param {Object} state     Data state.
  * @param {string} blockName Block type name.
  *
+ * @example
+ * ```js
+ * import { __, sprintf } from '@wordpress/i18n';
+ * import { store as blocksStore } from '@wordpress/blocks';
+ * import { useSelect } from '@wordpress/data';
+ *
+ * const ExampleComponent = () => {
+ *     const navigationBlockHasChildBlocksWithInserterSupport = useSelect( ( select ) =>
+ *         select( blocksStore ).hasChildBlocksWithInserterSupport(
```

Comment:
I'm not sure how popular this function is, but it would be good to explore what role it plays and maybe hide it from the documentation to avoid the noise.

Reply:
I really have no idea where it's used. An equivalent function is exported [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/blocks/src/api/registration.js#L565). Perhaps we should `@ignore` this one here? That might also be a good idea for all the data store selectors with wrappers in the `api` dir?